front/src/pages/nft/[id].tsx

```diff
@@ -34,7 +34,7 @@ const SingleNFTPage = (props: SingleNFTPageProps) => {
                     </div>
                     <NftCardInformation data={data}></NftCardInformation>
```

Comment:
you should handle situations when server returns 404 here, for now you could basically set up data?... for every place you are trying to map it

Reply:
and remove please closing tag here, even IDEA shows warning here
<NFTCardInformation .../>